From 07cc0a05957bb6cdbc038b1e517d55d3a75d1473 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 27 Sep 2018 20:40:10 +0200
Subject: [PATCH 1/1] efi_selftest: fix typos

fix typos
correct the header comment of efi_selftest_variables.c

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_selftest/efi_selftest_config_table.c | 12 ++++++------
 lib/efi_selftest/efi_selftest_controllers.c  |  2 +-
 lib/efi_selftest/efi_selftest_crc32.c        |  4 ++--
 lib/efi_selftest/efi_selftest_event_groups.c |  4 ++--
 lib/efi_selftest/efi_selftest_events.c       |  2 +-
 lib/efi_selftest/efi_selftest_exception.c    |  4 ++--
 lib/efi_selftest/efi_selftest_fdt.c          |  2 +-
 lib/efi_selftest/efi_selftest_textinput.c    |  2 +-
 lib/efi_selftest/efi_selftest_tpl.c          |  2 +-
 lib/efi_selftest/efi_selftest_variables.c    |  6 ++----
 lib/efi_selftest/efi_selftest_watchdog.c     |  2 +-
 11 files changed, 20 insertions(+), 22 deletions(-)

diff --git a/lib/efi_selftest/efi_selftest_config_table.c b/lib/efi_selftest/efi_selftest_config_table.c
index 2aa3fc7284..0bc5da6b0c 100644
--- a/lib/efi_selftest/efi_selftest_config_table.c
+++ b/lib/efi_selftest/efi_selftest_config_table.c
@@ -18,7 +18,7 @@ static efi_guid_t table_guid =
 		 0x17, 0x2e, 0x51, 0x6b, 0x49, 0x75);
 
 /*
- * Notification function, increments the notfication count if parameter
+ * Notification function, increments the notification count if parameter
  * context is provided.
  *
  * @event	notified event
@@ -33,23 +33,23 @@ static void EFIAPI notify(struct efi_event *event, void *context)
 }
 
 /*
- * Check crc32 of a table.
+ * Check CRC32 of a table.
  */
 static int check_table(const void *table)
 {
 	efi_status_t ret;
 	u32 crc32, res;
-	/* Casting from const to not const */
+	/* Casting from constant to not constant */
 	struct efi_table_hdr *hdr = (struct efi_table_hdr *)table;
 
 	crc32 = hdr->crc32;
 	/*
-	 * Setting the crc32 of the 'const' table to zero is easier than
+	 * Setting the CRC32 of the 'const' table to zero is easier than
 	 * copying
 	 */
 	hdr->crc32 = 0;
 	ret = boottime->calculate_crc32(table, hdr->headersize, &res);
-	/* Reset table crc32 so it stays constant */
+	/* Reset table CRC32 so it stays constant */
 	hdr->crc32 = crc32;
 	if (ret != EFI_ST_SUCCESS) {
 		efi_st_error("CalculateCrc32 failed\n");
@@ -203,7 +203,7 @@ static int execute(void)
 		return EFI_ST_FAILURE;
 	}
 	if (tabcnt > 1) {
-		efi_st_error("Duplicate table guid\n");
+		efi_st_error("Duplicate table GUID\n");
 		return EFI_ST_FAILURE;
 	}
 	if (table != &tables[1]) {
diff --git a/lib/efi_selftest/efi_selftest_controllers.c b/lib/efi_selftest/efi_selftest_controllers.c
index ceefa03444..e8a80d778d 100644
--- a/lib/efi_selftest/efi_selftest_controllers.c
+++ b/lib/efi_selftest/efi_selftest_controllers.c
@@ -33,7 +33,7 @@ static efi_handle_t handle_driver;
  * Count child controllers
  *
  * @handle	handle on which child controllers are installed
- * @protocol	protocol for which the child controlles where installed
+ * @protocol	protocol for which the child controllers were installed
  * @count	number of child controllers
  * @return	status code
  */
diff --git a/lib/efi_selftest/efi_selftest_crc32.c b/lib/efi_selftest/efi_selftest_crc32.c
index 8555b8f114..4881e8ac6f 100644
--- a/lib/efi_selftest/efi_selftest_crc32.c
+++ b/lib/efi_selftest/efi_selftest_crc32.c
@@ -5,7 +5,7 @@
  * Copyright (c) 2018 Heinrich Schuchardt <xypron.glpk@gmx.de>
  *
  * This unit test checks the CalculateCrc32 bootservice and checks the
- * headers of the system table, the boot services tablle, and the runtime
+ * headers of the system table, the boot services table, and the runtime
  * services table before and after ExitBootServices().
  */
 
@@ -19,7 +19,7 @@ static int check_table(const void *table)
 {
 	efi_status_t ret;
 	u32 crc32, res;
-	/* Casting from const to not const */
+	/* Casting from constant to not constant */
 	struct efi_table_hdr *hdr = (struct efi_table_hdr *)table;
 
 	if (!hdr->signature) {
diff --git a/lib/efi_selftest/efi_selftest_event_groups.c b/lib/efi_selftest/efi_selftest_event_groups.c
index 9b3c5132ef..5a7980c5d0 100644
--- a/lib/efi_selftest/efi_selftest_event_groups.c
+++ b/lib/efi_selftest/efi_selftest_event_groups.c
@@ -19,7 +19,7 @@ static efi_guid_t event_group =
 		 0x0e, 0x5b, 0x45, 0xc0, 0x56, 0x91);
 
 /*
- * Notification function, increments the notfication count if parameter
+ * Notification function, increments the notification count if parameter
  * context is provided.
  *
  * @event	notified event
@@ -114,7 +114,7 @@ static int execute(void)
 					      (unsigned int)i, (unsigned int)j,
 					      (unsigned int)counter[j]);
 				efi_st_error(
-					"Nofification function not called\n");
+					"Notification function not called\n");
 				return EFI_ST_FAILURE;
 			}
 		}
diff --git a/lib/efi_selftest/efi_selftest_events.c b/lib/efi_selftest/efi_selftest_events.c
index 47f9f99318..ed99a53804 100644
--- a/lib/efi_selftest/efi_selftest_events.c
+++ b/lib/efi_selftest/efi_selftest_events.c
@@ -17,7 +17,7 @@ static unsigned int timer_ticks;
 static struct efi_boot_services *boottime;
 
 /*
- * Notification function, increments the notfication count if parameter
+ * Notification function, increments the notification count if parameter
  * context is provided.
  *
  * @event	notified event
diff --git a/lib/efi_selftest/efi_selftest_exception.c b/lib/efi_selftest/efi_selftest_exception.c
index 2ac1796c3b..76cfb88d7c 100644
--- a/lib/efi_selftest/efi_selftest_exception.c
+++ b/lib/efi_selftest/efi_selftest_exception.c
@@ -4,7 +4,7 @@
  *
  * Copyright (c) 2018 Heinrich Schuchardt <xypron.glpk@gmx.de>
  *
- * Test the handling of execptions by trying to execute an undefined
+ * Test the handling of exceptions by trying to execute an undefined
  * instruction.
  */
 
@@ -37,7 +37,7 @@ static int execute(void)
 {
 	undefined_instruction();
 
-	efi_st_error("An undefined instruction exeption was not raised\n");
+	efi_st_error("An undefined instruction exception was not raised\n");
 
 	return EFI_ST_FAILURE;
 }
diff --git a/lib/efi_selftest/efi_selftest_fdt.c b/lib/efi_selftest/efi_selftest_fdt.c
index c7bc242b5c..d545d51812 100644
--- a/lib/efi_selftest/efi_selftest_fdt.c
+++ b/lib/efi_selftest/efi_selftest_fdt.c
@@ -16,7 +16,7 @@
 static struct efi_boot_services *boottime;
 static const char *fdt;
 
-/* This should be sufficent for */
+/* This should be sufficient for */
 #define BUFFERSIZE 0x100000
 
 static efi_guid_t fdt_guid = EFI_FDT_GUID;
diff --git a/lib/efi_selftest/efi_selftest_textinput.c b/lib/efi_selftest/efi_selftest_textinput.c
index 164fbffe6c..b90671cdd2 100644
--- a/lib/efi_selftest/efi_selftest_textinput.c
+++ b/lib/efi_selftest/efi_selftest_textinput.c
@@ -5,7 +5,7 @@
  * Copyright (c) 2018 Heinrich Schuchardt <xypron.glpk@gmx.de>
  *
  * Provides a unit test for the EFI_SIMPLE_TEXT_INPUT_PROTOCOL.
- * The unicode character and the scan code are printed for text
+ * The Unicode character and the scan code are printed for text
  * input. To run the test:
  *
  *	setenv efi_selftest text input
diff --git a/lib/efi_selftest/efi_selftest_tpl.c b/lib/efi_selftest/efi_selftest_tpl.c
index 61ae46b797..97d256abe4 100644
--- a/lib/efi_selftest/efi_selftest_tpl.c
+++ b/lib/efi_selftest/efi_selftest_tpl.c
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0+
 /*
- * efi_selftest_events
+ * efi_selftest_tpl
  *
  * Copyright (c) 2017 Heinrich Schuchardt <xypron.glpk@gmx.de>
  *
diff --git a/lib/efi_selftest/efi_selftest_variables.c b/lib/efi_selftest/efi_selftest_variables.c
index 146378fb9a..e4c389a872 100644
--- a/lib/efi_selftest/efi_selftest_variables.c
+++ b/lib/efi_selftest/efi_selftest_variables.c
@@ -4,10 +4,8 @@
  *
  * Copyright (c) 2018 Heinrich Schuchardt <xypron.glpk@gmx.de>
  *
- * This unit test checks the following protocol services:
- * ConnectController, DisconnectController,
- * InstallProtocol, ReinstallProtocol, UninstallProtocol,
- * OpenProtocol, CloseProtcol, OpenProtocolInformation
+ * This unit test checks the runtime services for variables:
+ * GetVariable, GetNextVariableName, SetVariable, QueryVariableInfo.
  */
 
 #include <efi_selftest.h>
diff --git a/lib/efi_selftest/efi_selftest_watchdog.c b/lib/efi_selftest/efi_selftest_watchdog.c
index bff2330918..cbc6761721 100644
--- a/lib/efi_selftest/efi_selftest_watchdog.c
+++ b/lib/efi_selftest/efi_selftest_watchdog.c
@@ -35,7 +35,7 @@ static struct notify_context notification_context;
 static bool watchdog_reset;
 
 /*
- * Notification function, increments the notfication count if parameter
+ * Notification function, increments the notification count if parameter
  * context is provided.
  *
  * @event	notified event
-- 
2.19.0

